File: js/repeatingLetters.js
// Create a function that takes a string and returns a string in which each character is repeated once.
// https://edabit.com/challenge/Mc6Xi4PRw7fDzeMDB

const doubleChar = (str) =>
  str
    .split("")
    .map((letter) => letter.repeat(2))
    .join("");

module.exports = doubleChar;

File: CSharp/RepeatingLetters.cs
namespace CSharp
{
    // Create a function that takes a string and returns a string in which each character is repeated once.
    // https://edabit.com/challenge/fKZHLzmR8anBrxgNt
    public static class RepeatingLetters
    {
        public static string DoubleChar(string str)
        {
            var result = "";

            for (var i = 0; i < str.Length; i++)
            {
                for (var j = 0; j < 2; j++)
                {
                    result += str[i];
                }
            }

            return result;
        }
    }
}